src/OpenAL/Extensions/Silk.NET.OpenAL.Extensions.Creative/Enums/BufferStorageMode.cs

```diff
@@ -6,7 +6,7 @@
 namespace Silk.NET.OpenAL.Extensions.Creative
 {
     /// <summary>
-    /// This enum is used to abstract the need of using AL.GetEnumValue() with the extension. The values do not
+    /// This enum is used to partial the need of using AL.GetEnumValue() with the extension. The values do not
```

Comment:
Lmao this one's my fault.
```suggestion
    /// This enum is used to abstract the need of using AL.GetEnumValue() with the extension. The values do not
```